meta/lifecycle.go

```diff
@@ -10,6 +10,10 @@ func LifeCycleFromBucket(b Bucket) (lc LifeCycle) {
 
 func (m *Meta) PutBucketToLifeCycle(bucket Bucket) error {
 	lifeCycle := LifeCycleFromBucket(bucket)
+	err := m.Client.RemoveBucketFromLifeCycle(bucket)
```

Comment:
it should be a transaction?

Reply:
Otherwise every time putLifecycle is inserted